src/ui/components/ResourceCell.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { Button, TableCell } from '@material-ui/core';
+
+const ResourceCell = ({ cellKey, resource }) => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div>
+      {!open && <Button onClick={() => setOpen(!open)}> open </Button>}
+      {open && (
+        <div>
+          <Button onClick={() => setOpen(!open)}> close</Button>
+          <TableCell key={cellKey}>
+            <pre>{JSON.stringify(resource, null, 2)}</pre>
```

Comment:
It would be really cool if you used something like https://www.npmjs.com/package/react-json-view here (this was just a quick google search but it seems easy enough to implement)